proposals/hybrid-rendering.md

```diff
@@ -0,0 +1,106 @@
+- Start Date: 2023-04-27
+- Reference Issues: https://github.com/withastro/roadmap/issues/539
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+Provide a new `output` option `'hybrid'` that treats all pages as prerendered by default, allowing an opt-out through `export const prerender = false`.
+
+# Example
+
+An existing static site can be changed to hybrid rendering, allowing some specific routes to not be prerendered.
+
+__astro.config.mjs__
+
+```js
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  output: 'hybrid'
+});
+```
```

Comment:
Suggesting this to make it clearer that hybrid requires an adapter
```suggestion
```diff
import { defineConfig } from 'astro/config';
import vercel from "@astrojs/vercel"

export default defineConfig({
-  output: 'static',
+  output: 'hybrid',
+  adapter: vercel()
});
```
```